src/net_processing.cpp

```diff
@@ -3306,6 +3312,19 @@ bool PeerLogicValidation::SendMessages(CNode* pto, std::atomic<bool>& interruptM
     return true;
 }
 
+void SetAutoRequestBlocks(bool state) {
+    fAutoRequestBlocks = state;
+}
+
+bool isAutoRequestingBlocks() {
+    return fAutoRequestBlocks;
+}
+
+unsigned int getAmountOfBlocksInFlight() {
```

Comment:
I think they should all start with capital or non-capital (`SetAuto` vs `isAuto`/`getAmount`) for consistency.

Reply:
Also nit, the braces should be on new lines